### PR TITLE
Joe final changes

### PR DESCRIPTION
**Pull request by joeShuff**

### CODE REVIEW

File: game/src/main/java/org/teamfarce/mirch/screens/InterviewScreen.java [hunk 1]
```diff
@@ -208,8 +236,50 @@ private void initInterviewStage() {
...
+        String room = "";
+        String weapon = "";
+
+        //Get the murder room name and the murder weapon
+        for (Room r : game.gameSnapshot.map.getRooms()) {
+            if (r.isMurderRoom()) {
+                room = r.getName();
+            }
+
+            for (Clue c : r.getClues()) {
```
Comment (jasonmash) [inline game/src/main/java/org/teamfarce/mirch/screens/InterviewScreen.java]: Is there not a nicer way of getting the means clue? Nested for loops are never a good thing. Also I don't understand why there are two for loops nested inside here, when they're both assigning to the same variable

**Comment by joeShuff** [comment]
So noone is awake, so I am going to do my tidy changes in this branch aswell.

**Comment by joeShuff** [comment]
closes #118 
closes #119 
closes #124 
closes #123 
closes #126 
closes #125 
closes #122

File: game/src/main/java/org/teamfarce/mirch/screens/InterviewScreen.java [hunk 2]
```diff
@@ -252,17 +313,30 @@ private void switchState(int result) {
...
         }
     }
 
+    /**
+     * Im not sure what this does, someone please add comment
+     *
+     * @param result
+     * @param clue
+     */
     private void questionClue(int result, Clue clue) {
```
Comment (joeShuff) [inline game/src/main/java/org/teamfarce/mirch/screens/InterviewScreen.java]: The doc string for this method needs doind, I'm not sure what it does

File: game/src/main/java/org/teamfarce/mirch/screens/InterviewScreen.java [hunk 3]
```diff
@@ -272,6 +346,10 @@ private void questionClue(int result, Clue clue) {
         }
     }
 
+    /**
+     * Im not sure what this does, someone please add comment
+     * @param result
+     */
     private void questionStyle(int result) {
```
Comment (joeShuff) [inline game/src/main/java/org/teamfarce/mirch/screens/InterviewScreen.java]: The doc string for this method needs doind, I'm not sure what it does

**Comment by joeShuff** [comment]
Okay I believe I have tidied everything up. Feel free to make comments/commits and merge it.

File: game/src/main/java/org/teamfarce/mirch/screens/InterviewScreen.java (hunk 2 above)
Comment (jasonmash) [inline game/src/main/java/org/teamfarce/mirch/screens/InterviewScreen.java]: I've sorted this out :)

**Pull request merged**